modules/secure-cloud-run-net/network.tf

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_compute_subnetwork" "vpc_subnetwork" {
+  count = var.create_subnet ? 1 : 0
+
+  name                       = var.subnet_name
+  project                    = var.vpc_project_id
+  network                    = var.shared_vpc_name
+  ip_cidr_range              = var.ip_cidr_range
+  region                     = var.location
+  private_ip_google_access   = true
+  private_ipv6_google_access = true
```

Comment:
we can remove this line, because our sub-net is IPV4 and this always trigger a Update in place.

Reply:
fixed